views/_partials/integrations.pug

```diff
@@ -1,9 +1,11 @@
 hr
 br
 a(href=item.url, target="_blank")
-    img.img-responsive.thumbnail.center-block(src=item.img, alt=item.name)
+    != helpers.cacheBuster.img(item.img, { alt: item.name, class: "img-responsive thumbnail center-block" })
 h3
     a(href=item.url, target="_blank")=item.name
-| Platform: 
```

Comment:
Right! Good catch, I have my EOL space removal set to ignore jade, need to update it to ignore pug too.